reconcile/utils/ocm.py

```diff
@@ -931,11 +931,37 @@ def _init_blocked_versions(self, blocked_versions):
                 raise TypeError(f"blocked version is not a valid regex expression: {b}")
 
     @retry(max_attempts=10)
-    def _get_json(self, api):
-        r = requests.get(f"{self.url}{api}", headers=self.headers)
+    def _do_get_request(self, api: str, params: dict[str, str]):
+        r = requests.get(f"{self.url}{api}", headers=self.headers, params=params)
         r.raise_for_status()
         return r.json()
 
+    @staticmethod
+    def _response_is_list(rs: dict[str, Any]) -> bool:
+        return rs["kind"].endswith("List")
+
+    def _get_json(self, api):
+        responses = []
+        params = {"size": 100}
+        while True:
+            rs = self._do_get_request(api, params=params)
+            responses.append(rs)
+            if self._response_is_list(rs) and rs["size"] == params["size"]:
```

Comment:
I haven't worked very closely with the OCM API, but I was just curious if the `List` suffix is always going to be general enough? I didn't see an obvious use of this in the `ocm-cli` linked from the ticket (100% possible I missed it).

Reply:
Yes, that's per definition: https://github.com/openshift-online/ocm-api-model